astor/rtrip.py

```diff
@@ -85,6 +85,10 @@
 dsttree = 'tmp_rtrip'
 
 
+def out_prep(s, pre_encoded=(type(b'') is type(''))):
```

Comment:
I might come up with an alternative way, but I don't fully understand the problem here. Care to elaborate on why do you want to check the results of `b'' == u''` and `type(b'') is type('')`? Do you want to know if the Python version is 2?